resources/adoptopenjdk_install.rb

```diff
@@ -0,0 +1,202 @@
+#
+# Cookbook:: java
+# Resource:: adoptopenjdk_install
+#
+# Based on oracle_install.rb by Bryan W. Berry (<bryan.berry@gmail.com>)
+#
+
+resource_name :adoptopenjdk_install
+
+default_action :install
+
+property :url, String
+property :checksum, String, regex: /^[0-9a-f]{32}$|^[a-zA-Z0-9]{40,64}$/
+property :md5, String, regex: /^[0-9a-f]{32}$|^[a-zA-Z0-9]{40,64}$/
```

Comment:
@tas50 should we use a `sha256` on this new resource or leave it to match the `oracle_install` resource?